### DISCUSSION TITLE
Moving `docs/requirements.txt` as a new extra-requirements section.

### DISCUSSION THREAD

**Discussion by abravalheri** [pyscaffold/pyscaffold#502] [Ideas]
I noticed in the `tox` [repository](https://github.com/tox-dev/tox/blob/a2370f4ad055cb5d570745d3a06016a7ea76d6cb/setup.cfg#L62) that instead of using a `docs/requirements.txt` like our templates do, they define these requirements in a optional [`docs` extra-requirements section](https://github.com/tox-dev/tox/blob/master/.readthedocs.yml).

When we move everything to `pyproject.toml`, should we also adopt this approach?

We are already doing this for the `testing` requirements btw, inspired by `tox` docs.

**Comment by FlorianWilhelm** [comment]
As a mathematician, I like when things converge ;-) So having it in `pyproject.toml` in a `docs` extra requirement sounds like the way to go for me.
